src/Database/Schema/Blueprint.php

```diff
@@ -0,0 +1,365 @@
+<?php
+
+namespace DesignMyNight\Elasticsearch\Database\Schema;
+
+use Carbon\Carbon;
+use Illuminate\Database\Connection;
+use Illuminate\Database\Schema\Grammars\Grammar;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Str;
+
+/**
+ * Class Blueprint
+ * @package DesignMyNight\Elasticsearch\Database\Schema
+ */
+class Blueprint extends \Illuminate\Database\Schema\Blueprint
```

Comment:
Are all the column types here ES-specific? I don't fully understand the 'range' ones

Reply:
Yeah I copied them from here: https://www.elastic.co/guide/en/elasticsearch/reference/6.7/mapping-types.html

Reply:
And the range sub types they have here: https://www.elastic.co/guide/en/elasticsearch/reference/6.7/range.html